juju/client/jujudata.py

```diff
@@ -77,7 +79,10 @@ def refresh(self):
 
     def current_controller(self):
         '''Return the current controller name'''
-        return self._load_yaml('controllers.yaml', 'current-controller')
+        try:
+            return self._load_yaml('controllers.yaml', 'current-controller')
+        except FileNotFoundError:
+            raise JujuControllerNotFoundError('No controllers.yaml file found. Did you forget to bootstrap Juju?')
```

Comment:
I'd change the wording here just a little bit, as people often deploy workloads that sets up the environment automatically (e.g. GH actions), so the part that sets up the juju env might fail even though they didn't forget to bootstrap themselves.

```suggestion
            raise JujuControllerNotFoundError('No controllers.yaml file found. python-libjuju requires a bootstrapped Juju controller.')
```